checker/checker.go

```diff
@@ -13,7 +13,6 @@ type Checker struct {
 	Timeout time.Duration
 
 	// Cache specifies the hostname scan cache store and expire time.
```

Comment:
```suggestion
	// Cache specifies the hostname scan cache store and expire time.
	// If `nil`, then scans are not cached.
```

Or a comment like this!